text/0000-use-group-as.md

```diff
@@ -0,0 +1,72 @@
+- Feature Name: use_group_as
+- Start Date: 2015-02-15
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Allow renaming imports when importing a group of symbols from a module.
+
+```rust
+use std::io::{
+    Error as IoError,
+    Result as IoResult,
+    Read,
+    Write
+}
+```
+
+# Motivation
+
+THe current design requires the above example to be written like this:
```

Comment:
`s/THe/The/` (unless it's intentional for some strange reason which I'm guessing it isn't)